lrs/managers/StatementManager.py

```diff
@@ -71,7 +73,11 @@ def save_statement_to_db(self):
             del self.data['id']
 
         # Try to create statement
-        stmt = Statement.objects.create(**self.data)
+        try:
+            stmt = Statement.objects.create(**self.data)
+        except IntegrityError:
```

Comment:
As much of a pain as it might be, do we want to check the error message itself for fragments mentioning it was due to a statement already existing with that ID - otherwise this behaviour might mask a different presentation of `IntegrityError` that we won't know to fix?

Reply:
Potentially. I'll have a look to see if any other integrity errors might be thrown